### PR TITLE
[ADD]account_invoice_discount_display_amount: Show total discount on invoices

### PR DESCRIPTION
**Pull request by manuelregidor**
This module shows the total discount amount and the amount without discount on invoices.

### CODE REVIEW

**Comment by manuelregidor** [comment]
@HaraldPanten @ValentinVinagre

File: invoice_discount_display_amount/models/account_move_line.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    discount_total = fields.Monetary(
+        compute="_compute_discount_amount", string="Discount Total", store=True
+    )
+    price_total_no_discount = fields.Monetary(
+        compute="_compute_discount_amount", string="Total Without Discount", store=True
+    )
+
+    @api.depends("discount", "price_total")
+    def _compute_discount_amount(self):
+        for line in self:
```
Comment (ValentinVinagre) [inline invoice_discount_display_amount/models/account_move_line.py]: You should only take into account those that are invoice lines

**Review by ValentinVinagre — CHANGES_REQUESTED** [review]

File: invoice_discount_display_amount/models/account_move_line.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            price_total_no_discount = 0
+            discount_total = 0
+            if not line.discount:
```
Comment (ValentinVinagre) [inline invoice_discount_display_amount/models/account_move_line.py]: can be added as a filtered in the for and make a massive change to those that are not invoice lines and do not have a discount

File: invoice_discount_display_amount/models/account_move.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        string="Total Without Discount",
+        currency_field="currency_id",
+        store=True,
+    )
+
+    @api.depends(
+        "invoice_line_ids.discount_total", "invoice_line_ids.price_total_no_discount"
+    )
+    def _compute_discount_total(self):
+        for invoice in self:
```
Comment (ValentinVinagre) [inline invoice_discount_display_amount/models/account_move.py]: you should filter those account.move that are not invoices and directly indicate that the value of the fields is 0.0 with a massive change.

**Review by ValentinVinagre — APPROVED** [review]
👯‍♂️

**Comment by ValentinVinagre** [comment]
@HaraldPanten could you review?

**Review by HaraldPanten — APPROVED** [review]
Functional review

**Comment by ValentinVinagre** [comment]
@AaronHForgeFlow could you review?

**Pull request merged**